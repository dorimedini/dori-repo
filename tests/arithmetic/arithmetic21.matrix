{

	matrix m1(3,3) = [1, 2, 3; 4, 5, 6; 7, 8, 9];
	matrix m2(3,3) = [3, 3, 3; 2, 2, 2; 1, 1, 1];
	matrix m3(3,2) = [1, 2; 4, 5; 7, 8];
	matrix m4(3,1) = [3; 6; 9];
	matrix m33(3,3);

	m33=m1 - m2 + m1;
	print m33;
	m33=m1 + m2 - m1;
	print m33;

		
}