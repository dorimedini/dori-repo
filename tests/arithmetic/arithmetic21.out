[-1,1,3;6,8,10;13,15,17]
[3,3,3;2,2,2;1,1,1]
